website/blog/2023-04-15-databend-weekly-89.md

```diff
@@ -0,0 +1,153 @@
+---
+title: "This Week in Databend #89"
+date: 2023-04-15
+slug: 2023-04-15-databend-weekly
+tags: [databend, weekly]
+description: "Stay up to date with the latest weekly developments on Databend!"
+contributors:
+  - name: andylokandy
+  - name: ariesdevil
+  - name: b41sh
+  - name: BohuTANG
+  - name: dependabot[bot]
+  - name: Dousir9
+  - name: everpcpc
+  - name: flaneur2020
+  - name: leiysky
+  - name: lichuang
+  - name: PsiACE
+  - name: RinChanNOWWW
+  - name: SkyFan2002
+  - name: soyeric128
+  - name: sundy-li
+  - name: TCeason
+  - name: Xuanwo
+  - name: xudong963
+  - name: youngsofun
+  - name: zhang2014
+  - name: zhyass
+authors:
+  - name: PsiACE
+    url: https://github.com/psiace
+    image_url: https://github.com/psiace.png
+---
+
+[Databend](https://github.com/datafuselabs/databend) is a modern cloud data warehouse, serving your massive-scale analytics needs at low cost and complexity. Open source alternative to Snowflake. Also available in the cloud: <https://app.databend.com> .
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+### Announcing Databend v1.1
+
+Databend v1.1.0 was officially released on April 14, 2023! This release marks the first major update of Databend since version 1.0.
+
+The new version has many new features, including:
+
+- `COPY INTO` now has ETL capabilities
+- Full Support for TPC-DS Queries
+- `REPLACE INTO`
+- Window Functions
+- Integration with Metabase, Redash and Grafana
+- Rust driver
+- AI Functions
+- AskBend - knowledge base Q&A system
+
+If you are interested in learning more, please check out the resources listed below.
+
+- [What's Fresh in Databend v1.1 | Blog | Databend](https://databend.rs/blog/databend-release-v1.1)
+
+### New AI-related Configurations
+
+Databend has added new AI-related configurations that allow for the use of different endpoints and models.
```

Comment:
Databend now offers additional configurations for AI-related tasks, allowing you the flexibility to choose from various endpoints and models.